styles/global.css

```diff
@@ -92,7 +95,7 @@ h5 {
 }
 
 small {
-  font-size: 0.8rem;
+  font-size: 0.8rem
```

Comment:
Semicolon missing?

Reply:
I think its some of @iharmanpannu 's CSS but resolving this in a commit below.